lib/ansible/modules/files/archive.py

```diff
@@ -30,7 +30,7 @@
   format:
     description:
       - The type of compression to use.
```

Comment:
please add a not about `xz`support was added in version 2.5
~~~diff
        - The type of compression to use.
+       - Support for xz was added in version 2.5.
~~~

Reply:
done